src/main/kotlin/apply/application/mail/MailSender.kt

```diff
@@ -1,5 +1,9 @@
 package apply.application.mail
 
+import org.springframework.core.io.ByteArrayResource
+
 interface MailSender {
     fun send(toAddress: String, subject: String, body: String)
+
+    fun sendBCC(toAddresses: Array<String>, subject: String, body: String, files: Map<String, ByteArrayResource>)
```

Comment:
단체 메일은 숨은 참조가 기본이므로 `send()` 또는 `sendBcc()` 중 하나를 선택하면 됩니다.
https://developer.android.com/kotlin/style-guide?hl=ko#camel_case

```suggestion
    fun sendBcc(toAddresses: Array<String>, subject: String, body: String, files: Map<String, ByteArrayResource>)
```